src/System.AppContext/src/System/AppContext.cs

```diff
@@ -44,6 +45,19 @@ public static object GetData(string name)
             return null;
         }
 
+        public static event UnhandledExceptionEventHandler UnhandledException
```

Comment:
FYI this implementation is .NET Native specific as the CoreCLR AppContext is in System.Private.CoreLib.